spectests/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    testonly = True,
+    srcs = [
+        "ssz_mainnet.yaml.go",
+        "ssz_minimal.yaml.go",
+    ],
+    importpath = "github.com/prysmaticlabs/go-ssz/spectests",
+    visibility = ["//visibility:public"],
+)
+
+go_test(
```

Comment:
don't forget to add the test size and tag spec tests with `manual` and `spectest`

Reply:
These aren't as expensive as I expected, we can omit `manual` but definitely add `spectest`